src/NetAppFiles/NetAppFiles/NetAppFiles.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
how is this line different?

Reply:
Hmmm, really can't say. I don't see any difference and have not made any change to it.

Reply:
Check if there are any white spaces, could be caused by your text editor.
Also simply fix the merge conflict, let me know if you have any update.
I'll try to merge it for you as soon as I can.

Reply:
I suspect it was caused by the editor but there is no whitespace difference.

Concerning the merge conflicts, because this is test record files and there are too many diffs to fix in github, I know of no other way than cloning from azure, force pushing to my fork, reapplying all the changes and then rerunning all the tests. It also means the PR is closed and a new one must be opened. This is time consuming and happens a lot while things are in review. Is there a shorter way? Thanks.

Reply:
Try rebasing: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md#rebasing

No need to close this or create a new PR, and no need to run all the tests by the way -- only the four tests that have conflicts need to be re-ran.